lessonbuilder/tool/src/webapp/js/show-page.js

```diff
@@ -193,6 +193,13 @@ $(document).ready(function () {
     }
   });
 
+  //Toggle collapsible section
+  $(".sectionHeader").attr("tabindex", "0").on("keydown", function (event) {
```

Comment:
If the markup can be adjusted, this would preferred vs some post load js correcting it.

Reply:
I have adjusted the markup and pushed the new code. Kindly review, thank you!